content/zh-cn/docs/contributor-guide.md

```diff
@@ -91,11 +91,22 @@ title: 贡献者指南
 [Microsoft Lens (Windows)](https://www.microsoft.com/en-us/p/office-lens/9wzdncrfj3t8)
 
 ### 附件目录
+
 根据规范，不同种类附件都有存放目录，建议将附件存放至规范的目录下。
 
-- 图片类附件建议建议存放至`/static/images/`目录下，在正文中使用 `/images/+路径` 的方式引用；
-- PDF等文档类附件建议存放至`/static/documents/`目录下，在正文中使用 `/documents/+路径` 的方式引用；
+- 图片类附件建议建议存放至 [`/static/images/`][images] 目录下，在正文中使用 `/images/+路径` 的方式引用；
+- PDF 等文档类附件建议存放至 [`/static/documents/`][documents] 目录下，在正文中使用 `/documents/+路径` 的方式引用；
+
+[images]: https://github.com/mtf-wiki/MtF-Wiki/tree/master/static/images/
+[documents]: https://github.com/mtf-wiki/MtF-Wiki/tree/master/static/documents/
 
 ### 表格
 
-表格类说明建议使用内嵌CSV格式，即首行使用` ‍```csv `，末行使用` ``` `，中间以逗号分隔符形式等内容填充。
+表格类说明建议使用内嵌 [CSV](https://zh.wikipedia.org/wiki/CSV) 格式
+
+````markdown
```

Comment:
see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks